src/api/lib/opensuse/validator.rb

```diff
@@ -84,7 +84,7 @@ def validate(opt, content)
         if File.exist?(schema_base_filename + '.rng')
           schema = Nokogiri::XML::RelaxNG(File.open(schema_base_filename + '.rng'))
         elsif File.exist?(schema_base_filename + '.xsd')
-          schema = Nokogiri::XML::Schema(File.open(schema_base_filename + '.xsd'))
+          schema = Nokogiri::XML::Schema(File.open(schema_base_filename + '.xsd'), Nokogiri::XML::ParseOptions.new.nononet)
```

Comment:
This change is not needed, our schemas do not include external entities, so network access for parsing is required. We are not doing it on master https://github.com/openSUSE/open-build-service/blob/50ef5823d66965dee95918f54e3d58fe70590bb4/src/api/app/lib/suse/validator.rb#L86 where we run the same version of nokogiri. It's not causing issues if we enable it, but I would stick with the defaults unless its needed